setup.py

```diff
@@ -51,7 +51,8 @@
         'ftw.profilehook',
         'Pillow',
         'ftw.colorbox',
-        'plone.api'
+        'plone.api',
+        'plone.batching>=1.0.4'
```

Comment:
@maethu I don't know anymore. Is that a problem? I can test it if it works for earlier versions.